src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import core.basesyntax.models.*;
+import java.util.Random;
+
+class FigureSupplier {

```

Comment:
Don't begin class or method implementation with an empty line. Remove the redundant empty line at the start of the class definition.